build-system/npm-publish/write-package-files.js

```diff
@@ -42,6 +58,22 @@ async function writePackageJson() {
     return;
   }
 
+  const exports = {
+    '.': './preact',
+    './preact': {
+      import: './dist/component-preact.module.js',
+      require: './dist/component-preact.js',
+    },
+    './react': {
+      import: './dist/component-react.module.js',
+      require: './dist/component-react.js',
+    },
+  };
+
+  if (await hasStylesheets()) {
```

Comment:
Doesn't this need `ext, version` as params?

Reply:
`extension` and `extensVersion` are globals in this script, see `const [extension, ampVersion, extensionVersion] = process.argv.slice(2);` at the top.

Reply:
Thank you for the explanation! I missed that context